components/docs-chef-io/content/automate/chef-automate-ha-prerequisites.md

```diff
@@ -0,0 +1,205 @@
++++
+title = "On-Premise Prerequisites"
+
+draft = false
+
+gh_repo = "On-Premise Prerequisites"
+[menu]
+  [menu.automate]
+    title = "On-Premise Prerequisites"
+    parent = "automate/deploy_high_availability"
+    identifier = "automate/settings/chef-automate-ha-prerequisites.md On-Premise Prerequisites"
+    weight = 12
++++
+
+{{< warning >}}
+{{% automate/ha-warn %}}
+{{< /warning >}}
+
+{{< warning >}}
+The below pre-requisites are according to our organizational standard. If you are using any specified version not mentioned here or a third party extensions or software you can reach out to the CAMs and our Customer Support Team.
+{{< /warning >}}
+
+Before installing Chef automate HA on On-premise deployment, ensure you have taken a quick tour of this pre-requisite page.
```

Comment:
`Before installing Chef automate HA on On-premise deployment` can you rewrite this sentence plz